custom/icds/messaging/indicators.py

```diff
@@ -35,6 +36,36 @@
 
 DEFAULT_LANGUAGE = 'hin'
 
+REPORT_IDS = [
+    HOME_VISIT_REPORT_ID,
+    THR_REPORT_ID,
+    CHILDREN_WEIGHED_REPORT_ID,
+    DAYS_AWC_OPEN_REPORT_ID,
+]
+
+
+@quickcache(['domain'], timeout=4 * 60 * 60, memoize_timeout=4 * 60 * 60)
+def get_report_configs(domain):
+    app = get_app(domain, SUPERVISOR_APP_ID)
```

Comment:
i'm not sure i know enough about mobile ucr to answer that - pinging @emord since he worked on this initially.

it looks like what we're doing is inspecting the ReportModules of the app and getting the report configs out of it. as long as that all still works correctly with the latest starred build, i think using the latest starred build sounds right to me.

@snopoke @emord does that sound right to you?